<launch>
  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
  </group>
  <include file="$(find project_turtleplan)/launch/bringTurtles/bring_onerobot.launch" >
    <arg name="init_pose_x" value="0" />
    <arg name="init_pose_y" value="0"/>
    <arg name="init_pose_a" value="0"/>
    <arg name="robot_name"  value="robot1" />
  </include>
  <!-- BEGIN ROBOT 2 -->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
  </group>
  <include file="$(find project_turtleplan)/launch/bringTurtles/bring_onerobot.launch" >
    <arg name="init_pose_x" value="0" />
    <arg name="init_pose_y" value="-2"/>
    <arg name="init_pose_a" value="0"/>
    <arg name="robot_name"  value="robot2" />
  </include>
  <!-- BEGIN ROBOT 3 -->
  <group ns="robot3">
    <param name="tf_prefix" value="robot3_tf" />
  </group>
  <include file="$(find project_turtleplan)/launch/bringTurtles/bring_onerobot.launch" >
    <arg name="init_pose_x" value="0" />
    <arg name="init_pose_y" value="-4"/>
    <arg name="init_pose_a" value="0"/>
    <arg name="robot_name"  value="robot3" />
  </include>
</launch>